specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -130,7 +130,7 @@
           "default": {
             "description": "DefaultErrorResponse",
             "schema": {
-              "$ref": "#/definitions/ErrorDetails"
+              "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
```

Comment:
There are still many instances of default. Given this is stable version, you might want to hold off just changing this one. If you do want to do a breaking change on this, should scrub all of the default and remove your own ErrorDetail and ErrorResponse defintion all together.